src/main/java/org/example/taskify/dto/req/ProjectReqDto.java

```diff
@@ -0,0 +1,12 @@
+package org.example.taskify.dto.req;
+
+import java.time.LocalDate;
+import lombok.Data;
+
+@Data
+public class ProjectReqDto {
```

Comment:
Don’t  use abbreviations 
```suggestion
public class ProjectRequestDto {
```